torch/utils/_pytree.py

```diff
@@ -290,6 +290,18 @@ def register_dataclass(cls: type[Any]) -> None:
     torch.export.register_dataclass(cls)
 
 
+def _constant_flatten(x):  # type: ignore[no-untyped-def]
+    return [], ConstantNode(x)
+
+
+def _constant_unflatten(_, context):  # type: ignore[no-untyped-def]
+    return context.value
+
+
+def _constant_flatten_with_keys(x):  # type: ignore[no-untyped-def]
+    return [], ConstantNode(x)
```

Comment:
```suggestion
def _constant_flatten(x: Any) -> tuple[list[Any], Context]:
    return [], x


def _constant_unflatten(_: Iterable[Any], context: Context) -> Context:
    return context


def _constant_flatten_with_keys(x: Any) -> tuple[list[KeyEntry], Context]:
    return [], x
```

It is unclear why we need to wrap with an unnecessary `ConstantNode(...)`.

Reply:
I put the ConstantNode there originally because it was clearer what is going on when you look at a TreeSpec